addons/osio-addon/pom.xml

```diff
@@ -20,6 +20,10 @@
       <groupId>io.fabric8.launcher</groupId>
       <artifactId>launcher-core-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
```

Comment:
@gastaldi  We actually already have a library available to us which has this capability: String org.codehaus.plexus.util.FileUtils.extension(String filename). Perhaps it's better to use what we already have instead of adding a new library?

Reply:
That class is part of the Maven integration, I'd rather use commons-io for this instead

Reply:
sure